src/Tools/dotnet-dump/Dumper.Windows.cs

```diff
@@ -7,16 +7,21 @@
 using System.Diagnostics;
 using System.IO;
 using System.Runtime.InteropServices;
-using System.Threading.Tasks;
 
 namespace Microsoft.Diagnostics.Tools.Dump
 {
     public partial class Dumper
     {
         private static class Windows
         {
-            internal static void CollectDump(Process process, string outputFile, DumpTypeOption type)
+            internal static void CollectDump(int processId, string outputFile, DumpTypeOption type)
             {
+                using SafeProcessHandle processHandle = NativeMethods.OpenProcess(NativeMethods.PROCESS_QUERY_INFORMATION | NativeMethods.PROCESS_VM_READ, false, processId);
```

Comment:
Can we add a comment that explains why we are deliberately not using System.Diagnostics.Process? Otherwise someone in the future might be tempted to 'simplify the code' right back to what it was : )